swift_static_framework/swift_static_framework.bzl

```diff
@@ -33,9 +33,24 @@ def _zip_generated_objc_hdr_arg(module_name, generated_objc_hdr_file):
         file_path = generated_objc_hdr_file.path,
     )
 
+def _zip_modulemap_arg(module_name, modulemap_file):
+    return "{module_name}.framework/Modules/module.modulemap={file_path}".format(
+        module_name = module_name,
+        file_path = modulemap_file.path,
+    )
+
+def _modulemap_file_content(module_name):
+    return """\
+framework module %s {{
+  header %s-Swift.h
```

Comment:
I think this is invalid now and you'll need to use `{module}` and then change the format to `module=module_name`

Reply:
Oops